devops/ci/multi_node_deploy/3node_destroy.jenkinsfile

```diff
@@ -122,26 +122,31 @@ node {
 
             stage("${remote.name}: Uninstall rpms") {
                 sshCommand remote: remote, command: """
+                    yum erase -y cortx-prvsnr cortx-prvsnr-cli      # Cortx Provisioner packages
+                    # Remove cortx-py-utils
+                    pip3 uninstall -y cortx-py-utils
+                    yum erase -y glusterfs-fuse glusterfs-server glusterfs     # Gluster FS packages
                     yum erase -y salt-minion salt-master salt-api   # Salt packages
                     yum erase -y python36-m2crypto                  # Salt dependency
                     yum erase -y python36-cortx-prvsnr              # Cortx Provisioner API packages
+                    # Brute clean for any cortx rpm packages
                     yum erase -y *cortx*
-                    yum autoremove -y
-                    yum clean all
-                    rm -rf /var/cache/yum
-                    # Remove cortx-py-utils
-                    pip3 uninstall -y cortx-py-utils
                     # Cleanup pip packages
                     pip3 freeze|xargs pip3 uninstall -y
-                    # Cleanup pip config
-                    test -e /etc/pip.conf && rm -f /etc/pip.conf
-                    test -e ~/.cache/pip && rm -rf ~/.cache/pip || true
+                    yum remove -y python3
```

Comment:
Remove this line. It's not required and risky.

Reply:
Ok